libcodechecker/libhandlers/store.py

```diff
@@ -100,6 +118,20 @@ def add_arguments_to_parser(parser):
                              "the '--name' parameter given to 'codechecker-"
                              "analyze' will be used, if exists.")
 
+    parser.add_argument('--host',
```

Comment:
We should introduce an `argument_group` for these arguments and explain that they are used to specify where the resultes are `store`d.